pokepro/src/components/Loader/Loader.tsx

```diff
@@ -1,11 +1,11 @@
-import React from 'react'
-
+import React from "react";
+import pokeballLoader from "../../assets/assets/pokeball-loader.gif";
 function Loader() {
   return (
```

Comment:
Return statement is not required here, also add in Reac.FC

```suggestion
const Loader: React.FC = () => (
```